egui_demo_lib/src/backend_panel.rs

```diff
@@ -208,6 +208,10 @@ impl BackendPanel {
             if ui.button("Quit").clicked() {
                 frame.quit();
             }
+
+            if ui.button("Drag Window").is_pointer_button_down_on() {
```

Comment:
```suggestion
            if ui.button("Drag me to drag window").is_pointer_button_down_on() {
```

or perhaps a tooltip